main/newscorm/learnpath.class.php

```diff
@@ -5648,13 +5660,15 @@ public function return_new_tree($update_audio = 'false', $drop_element_here = fa
 
             if ($is_allowed_to_edit) {
                 if (!$update_audio || $update_audio <> 'true') {
-                    $move_icon .= '<a class="moved" href="#">';
-                    $move_icon .= Display::return_icon('move_everywhere.png', get_lang('Move'), array(), ICON_SIZE_TINY);
-                    $move_icon .= '</a>';
+                    if ($arrLP[$i]['item_type'] !== TOOL_FINAL_ITEM) {
```

Comment:
This requires some code documentation: "// If it's a final item, we don't want the user to move it around, so hide the option".
Something like that.